core/templates/dev/head/css/oppia.css

```diff
@@ -3951,3 +3951,10 @@ span.oppia-issues-learner-action {
   width: 100%;
   border: 1px solid black;
 }
+
+/*To fix the shifting of tooltip when text is too long*/
```

Comment:
Add spaces after `/*` and before `*/`. End sentences with a period. Drop the newline below this line so that the comment is associated with the rule.